.github/workflows/release-discord-notify.yml

```
@@ -0,0 +1,27 @@
+name: release-discord-notify
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  notify:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Discord Notify
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: ${{ secrets.OKP4_DISCORD_WEBHOOK }}
+          method: 'POST'
+          customHeaders: '{"Content-Type": "application/json"}'
+          data: '{
+              "avatar_url": "https://avatars.githubusercontent.com/u/98603954?v=4",
+              "username": "Bot Anik",
+              "content": "
+          🚨 A new version of @${{github.repository}} ${{ github.event.tagName }} has been released! 🎉
+
+          👉 Changelog: https://github.com/${{ github.repository }/releases/tag/${{ github.event.tagName }}
+          👉 Documentation: https://ui.okp4.space/?path=/story/welcome--page
+          👉 Official repo: https://github.com/${{ github.repository }
+          👉 NPM link: https://www.npmjs.com/package/@okp4/ui
```

You should parametrize this as well (in case of copy&paste, it can avoid mistakes).

```suggestion
          👉 NPM link: https://www.npmjs.com/package/@${{ github.repository }
```